klantcontactmomenten/src/test/kotlin/nl/nlportal/klant/contactmomenten/service/impl/KlantContactMomentenServiceImplTest.kt

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.nlportal.klant.contactmomenten.service.impl
+
+import com.ritense.portal.commonground.authentication.CommonGroundAuthentication
+import com.ritense.portal.commonground.authentication.JwtBuilder
+import com.ritense.portal.klant.client.OpenKlantClient
+import com.ritense.portal.klant.domain.klanten.Klant
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runBlockingTest
+import nl.nlportal.klant.contactmomenten.client.KlantContactMomentenClient
+import nl.nlportal.klant.contactmomenten.domain.ContactMoment
+import nl.nlportal.klant.generiek.domain.ResultPage
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNull
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import org.mockito.Mock
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+import org.mockito.MockitoAnnotations
+
+@ExperimentalCoroutinesApi
+internal class KlantContactMomentenServiceImplTest {
+
+    @Mock
+    lateinit var klantContactMomentenClient: KlantContactMomentenClient
+
+    @Mock
+    lateinit var klantClient: OpenKlantClient
+
+    lateinit var klantContactMomentenServiceImpl: KlantContactMomentenServiceImpl
+
+    @BeforeEach
+    fun setUp() {
+        MockitoAnnotations.openMocks(this)
+        klantContactMomentenServiceImpl = KlantContactMomentenServiceImpl(klantContactMomentenClient, klantClient)
+    }
+
+    @Test
+    @Disabled
```

Comment:
Can you explain why these tests are `@Disabled`?

Reply:
Because locally without @Disabled all the tests run fine, but when run in the pipeline it fails with a exception: KlantContactMomentenServiceTest > get klantcontactmomenten  with unknown Authentication() FAILED
 org.mockito.exceptions.misusing.UnfinishedVerificationException at KlantContactMomentenServiceTest.kt:52

Reply:
I will check, because I dislike disabling tests for reasons that should be fixable.

Reply:
I also dislike this, but I could not find the reason

Reply:
I couldn't reproduce this yet. Do you have a log from the build that fails?

Reply:
https://github.com/nl-portal/nl-portal-backend-libraries/actions/runs/6183611184

Reply:
I'm inclined to accept it for now, but we should still fix it in the near future. It's probably an misconfiguration in the build pipeline.

Reply:
@theo-ritense https://github.com/nl-portal/nl-portal-backend-libraries/actions/runs/6172155548/job/16751848317